recipe/meta.yaml

```diff
@@ -1,20 +1,21 @@
 {% set name = "qh3" %}
-{% set version = "1.4.5" %}
+{% set version = "1.5.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/qh3-{{ version }}.tar.gz
-  sha256: 62150b050b3378827554e34916b8e8088cd676e974c9c5c02eed4c5c5472143c
+  sha256: 3d2e1616023286373ab40a276cbf0b8960e6bda640bca0569495dad57775f6b8
 
 build:
   skip: true  # [py<37]
   script:
     - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
     - export OPENSSL_DIR=$PREFIX  # [osx]
     - export AWS_LC_SYS_CMAKE_BUILDER=1
+    - export CMAKE_C_COMPILER=gcc  # [not win]
```

Comment:
`gcc` is not the right one, try this:
```suggestion
    - export CMAKE_C_COMPILER=${CC}  # [not win]
```

Reply:
Ah! I tried something like that earlier but was missing the `$`, thanks